corehq/apps/data_dictionary/views.py

```diff
@@ -272,6 +272,22 @@ def data_dictionary_json_case_properties(request, domain, case_type_name):
             "properties": []
         })
 
+    # properties_queryset skips groups with no properties. Add them here
+    empty_groups = (
+        CasePropertyGroup.objects
+        .annotate(properties_count=Count('property'))
```

Comment:
What is this being used for? When looping through the groups returned from this query it doesn't seem like `properties_count` is being used at all.

Reply:
It's used two lines down, on line 280, to select only the groups with 0 case properties.